File: src/main/java/dev/lonami/uniffidl/psi/UdlTokenType.java
package dev.lonami.uniffidl.psi;

import com.intellij.psi.tree.IElementType;
import dev.lonami.uniffidl.UdlLanguage;
import org.jetbrains.annotations.NonNls;
import org.jetbrains.annotations.NotNull;

public class UdlTokenType extends IElementType {
    public UdlTokenType(@NonNls @NotNull String debugName) {
        super(debugName, UdlLanguage.INSTANCE);
    }

    @Override
    public String toString() {
        return "UdlTokenType." + super.toString();
    }
}

File: src/main/kotlin/dev/lonami/uniffidl/psi/UdlTokenType.kt
package dev.lonami.uniffidl.psi

import com.intellij.psi.tree.IElementType
import dev.lonami.uniffidl.UdlLanguage
import org.jetbrains.annotations.NonNls

class UdlTokenType(@NonNls debugName: String) : IElementType(debugName, UdlLanguage.INSTANCE) {
    override fun toString(): String = "UdlTokenType.${super.toString()}"
}
